src/backend/api/meals.js

```diff
@@ -2,14 +2,72 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
+//select all
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    const meals = await knex("meal").select("*");
+    response.json(meals);
+  } catch (error) {
+    throw error;
```

Comment:
Any specific reason why you're catching and then throwing the error? Removing the `try .. catch` block would result in the same thing.